docs/resources/samples/convert-csv.md

```diff
@@ -24,39 +24,46 @@ Add the following script and build a flow using the steps given to try the sampl
 ## Sample code: Insert comma-separated values into a workbook
 
 ```TypeScript
+/**
+ * Convert incoming CSV data into a range and add it to the workbook.
```

Comment:
CSV is a complex file format. Does this support all CSV oddities?

One recommendation is to use the TSV file format, when possible, to avoid the difficulty of properly parsing a CSV.

Reply:
We're not making this sample to promote CSV or TSV. Many 3rd party services provide data as CSV files. This has also been a common scenario from customers.